runtime/src/main/java/io/quarkiverse/cxf/graal/CXFSubstitutions.java

```diff
@@ -76,7 +76,7 @@ public WrapperHelper compile(Class<?> wrapperType, Method[] setMethods,
             Method[] getMethods, Method[] jaxbMethods,
             Field[] fields, Object objectFactory) {
         Logger LOG = LogUtils.getL7dLogger(Target_org_apache_cxf_jaxb_WrapperHelperClassGenerator.class);
-        LOG.info("compileWrapperHelper substitution");
+        LOG.fine("compileWrapperHelper substitution");
```

Comment:
I think `debug` is more common. But it's a matter of taste...

Reply:
This one is actually not a `org.jboss.logging.Logger`, instead it is [`java.util.logging.Logger`](https://docs.oracle.com/en/java/javase/11/docs/api/java.logging/java/util/logging/Logger.html) ... there is no debug or debugf method.

Reply:
Ah, good point.